Capstone_Project/BaseLine.ipynb

```diff
@@ -1,5783 +0,0 @@
-       "    <tr>\n",
-       "      <th>3</th>\n",
-       "      <td>0</td>\n",
-       "      <td>3</td>\n",
-       "      <td>2</td>\n",
-       "      <td>Tōkyō-to Chiyoda-ku Kudanminami</td>\n",
-       "      <td>Italian/French</td>\n",
-       "      <td>35.694003</td>\n",
-       "      <td>139.753595</td>\n",
-       "      <td>0</td>\n",
-       "      <td>4</td>\n",
-       "      <td>22</td>\n",
-       "      <td>1</td>\n",
-       "      <td>2016</td>\n",
-       "      <td>1</td>\n",
-       "      <td>0</td>\n",
-       "      <td>0</td>\n",
-       "      <td>175.447598</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>4</th>\n",
-       "      <td>0</td>\n",
-       "      <td>6</td>\n",
-       "      <td>5</td>\n",
-       "      <td>Tōkyō-to Chiyoda-ku Kudanminami</td>\n",
-       "      <td>Italian/French</td>\n",
-       "      <td>35.694003</td>\n",
-       "      <td>139.753595</td>\n",
-       "      <td>0</td>\n",
-       "      <td>4</td>\n",
-       "      <td>29</td>\n",
-       "      <td>1</td>\n",
-       "      <td>2016</td>\n",
-       "      <td>1</td>\n",
-       "      <td>0</td>\n",
-       "      <td>0</td>\n",
-       "      <td>175.447598</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
-      ],
-      "text/plain": [
-       "   air_store_id  time_diff  visitors                    air_area_name  \\\n",
-       "0             0          3         2  Tōkyō-to Chiyoda-ku Kudanminami   \n",
-       "1             0          6         4  Tōkyō-to Chiyoda-ku Kudanminami   \n",
-       "2             0          3         2  Tōkyō-to Chiyoda-ku Kudanminami   \n",
-       "3             0          3         2  Tōkyō-to Chiyoda-ku Kudanminami   \n",
-       "4             0          6         5  Tōkyō-to Chiyoda-ku Kudanminami   \n",
-       "\n",
-       "   air_genre_name   latitude   longitude  holiday_flg  Week_day  day  month  \\\n",
-       "0  Italian/French  35.694003  139.753595            0         3   14      1   \n",
-       "1  Italian/French  35.694003  139.753595            0         4   15      1   \n",
-       "2  Italian/French  35.694003  139.753595            0         5   16      1   \n",
-       "3  Italian/French  35.694003  139.753595            0         4   22      1   \n",
-       "4  Italian/French  35.694003  139.753595            0         4   29      1   \n",
-       "\n",
-       "   year  fri  sat  sun  geo_location  \n",
-       "0  2016    0    0    0    175.447598  \n",
-       "1  2016    1    0    0    175.447598  \n",
-       "2  2016    0    1    0    175.447598  \n",
-       "3  2016    1    0    0    175.447598  \n",
-       "4  2016    1    0    0    175.447598  "
-      ]
-     },
-     "execution_count": 236,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "df_total.head()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 237,
-   "m
```

Comment:
@nishalpattan : what is the RMSE for the test set?

Reply:
@nishalpattan : Perhaps not for this PR and more for you to check later but it would be worth for you to check concepts on k-fold cross-validation. I would suggest to do some googling but perhaps you could start with :
- https://en.wikipedia.org/wiki/Cross-validation_(statistics)
- http://scikit-learn.org/stable/modules/cross_validation.html

Reply:
@nishalpattan : I think that a visually informative way of showing the performance of the mode would be to plot predicted vs. actual values in a scatter plot. What do you think?